perspective.el

```diff
@@ -196,54 +166,54 @@ Run with the activated perspective active.")
   (substitute-key-definition 'perspective-map nil persp-mode-map)
   (define-key persp-mode-map newkey 'perspective-map))
 
-;; make-variable-frame-local is obsolete according to the docs,
-;; but I don't want to have to manually munge frame-parameters
-;; all the time so I'm using it anyway.
-(make-variable-frame-local
- (defvar perspectives-hash nil
-   "A hash containing all perspectives. The keys are the
-perspectives' names. The values are persp structs,
-with the fields NAME, WINDOW-CONFIGURATION, BUFFERS,
-BUFFER-HISTORY, KILLED, POINT-MARKER, and LOCAL-VARIABLES.
+;; Keeping the variable definitions here since the docstrings might
+;; still be useful for developers.
```

Comment:
Just get rid of these; this is why Git history exists.

Reply:
I've removed the commented out variables.